pkg/providers/imagefamily/bootstrap/aksbootstrap.go

```diff
@@ -145,32 +145,19 @@ var (
 	// NodeTemplate                : user input that could be per-node (t) - ALL DEFAULTED FOR NOW
 	// Provisioner spec            : selected nodepool-level user input (p)
 
-	// NodeBootstrapVariables carries all variables needed to bootstrap a node
+	// staticNodeBootstrapVars carries all variables needed to bootstrap a node
 	// It is used as input rendering the bootstrap script Go template (customDataTemplate)
 	// baseline, covering unused (-), static (s), and unsupported (n) fields,
 	// as well as defaults, cluster/node level (cd/td/xd)
 	staticNodeBootstrapVars = nbcontractv1.Configuration{
```

Comment:
One sub-goal is to reduce the initialization of contract variables from bootstrapper by defaulting so I removed some of them here. We should be able to remove more later.